doc/classes/ProjectSettings.xml

```diff
@@ -73,7 +73,8 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Converts a localized path ([code]res://[/code]) to a full native OS path.
+				Converts a localized path ([code]res://[/code]) to a full native OS path. 
+				[b]Note:[/b] This method doesn't work in an exported project. To load files outside the PCK in an exported project, append the file path to the result of [method OS.get_executable_path] instead.
```

Comment:
> This method doesn't work in an exported project.

It should work with `usr:\\` paths, if it's not it's a bug (`usr:\\` files are located under `OS.get_user_data_dir`).

Reply:
Also, on macOS, one should use `OS.get_bundle_resource_dir()` instead of executable path, to get "NAME.app/Contents/Resources/", PCK is placed in this folder too. Executable path will work for the main executable itself and dynamic libs (though it more correct to place libs in the Frameworks), but placing files of any other type alongside executable will render macOS bundle invalid (it will work, but it will be impossible to sign and notarize).